### DISCUSSION TITLE
Account deletion feature?

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#1563] [Ideas] [answered]
It would be nice to have the option to delete your monkey type account :>

**Comment by Otard95** [comment, marked as answer]
Really its required by GDPR, as well as the option to download your own data.

**Reply by Miodec** [reply]
They are both available on request, either on Discord or by email

**Reply by ghost** [reply]
> They are both available on request, either on Discord or by email

Is it possible to add a button on the website to delete an account though?

**Reply by Otard95** [reply]
> Is it possible to add a button on the website to delete an account though?

One option could be to have a button or form which tells the Discord bot to send the request to a mod/admin channel. Though im not sure how the bot integrates to the website so i dont know how feasible it is.

**Reply by Miodec** [reply]
There will be also a button available in the settings in a future update.

**Reply by ghost** [reply]
> There will be also a button available in the settings in a future update.

When do you think it'll be available :>

**Reply by Miodec** [reply]
I don't know, after i move onto the new database - but no estimates on that. If you need to delete an account send me an email.

**Reply by Kalashnikeforce** [reply]
asd
